library/src/main/java/com/github/paolorotolo/appintro/AppIntroViewPager.java

```diff
@@ -140,7 +140,7 @@ private boolean checkCanRequestNextPage(MotionEvent event) {
     }
 
     private void checkIllegallyRequestedNextPage(MotionEvent event) {
-        int swipeThreshold = 25;
+        int swipeThreshold = 50;
```

Comment:
Go ahead and define this as a constant to this file -- make use of that here and on `detectSwipeToEnd()`